xpdan/tests/conftest.py

```diff
@@ -55,9 +55,11 @@ def build_pymongo_backed_broker(request):
     db_name = "mds_testing_disposable_{}".format(str(uuid.uuid4()))
     mds_test_conf = dict(database=db_name, host='localhost',
                          port=27017, timezone='US/Eastern')
-    mds = MDS(mds_test_conf, 1,
-              # auth=False
-              )
+    try:
+       # nasty details: to save MacOS user
+        mds = MDS(mds_test_conf, 1, auth=False)
+    except:
```

Comment:
I would put a `TypeError` or some other exception here, just so we don't have a super broad exception.